capstone/src/main/webapp/youtubeScript.js

```diff
@@ -0,0 +1,137 @@
+const ATTRIBUTES_BY_LANGUAGE = {
+  'en': ['TOXICITY', 'SEVERE_TOXICITY', 'TOXICITY_FAST', 'IDENTITY_ATTACK', 'INSULT', 'PROFANITY', 'THREAT', 'SEXUALLY_EXPLICIT', 'FLIRTATION'],
+  'es': ['TOXICITY', 'SEVERE_TOXICITY', 'IDENTITY_ATTACK_EXPERIMENTAL', 'INSULT_EXPERIMENTAL', 'PROFANITY_EXPERIMENTAL', 'THREAT_EXPERIMENTAL'],
+  'fr': ['TOXICITY', 'SEVERE_TOXICITY', 'IDENTITY_ATTACK_EXPERIMENTAL', 'INSULT_EXPERIMENTAL', 'PROFANITY_EXPERIMENTAL', 'THREAT_EXPERIMENTAL'],
+  'de': ['TOXICITY', 'SEVERE_TOXICITY', 'IDENTITY_ATTACK', 'INSULT', 'PROFANITY', 'THREAT'],
+  'it': ['TOXICITY', 'SEVERE_TOXICITY', 'IDENTITY_ATTACK', 'INSULT', 'PROFANITY', 'THREAT'],
+  'pt': ['TOXICITY', 'SEVERE_TOXICITY', 'IDENTITY_ATTACK', 'INSULT', 'PROFANITY', 'THREAT']
+};
+
+/** Calls youtube servlet and passes output to perspctive */
+async function callYoutube() {
+  const channelId = document.getElementById('channelIdForAnalysis').value;
+  if (!channelId) {
+    return;
+  }
+  const response = await fetch('/youtube_servlet?channelId=' + channelId,);
+  const comments = await response.json();
+  const commentListElement = document.getElementById('comment-list');
+  commentListElement.innerHTML = '';
+  const requestedAttributes = await getRequestedAttributes();
```

Comment:
I think we can remove the `await` here and then also the `async` from the `getRequestedAttributes()`

Reply:
+1

Reply:
addressed in "address nits"